### DISCUSSION TITLE
I expect that when using docker deployment, multiple music file directories can be selected

### DISCUSSION THREAD

**Discussion by pdone** [navidrome/navidrome#2657] [Ideas]
I expect that when using docker deployment, multiple music file directories can be selected.
And in the user interface, I can control whether these directories are displayed in the music library.

**Comment by metalheim** [comment]
#192 an issue already exists
Until this feature is developed, it isnt possible.

As a workaround, with docker, you can bind mount multiple volumes into the container musicfolder, so that navidrome sees the files of all of them.
All users will have access to All folders this way though.

This is what I am using as a workaround at this moment
